peers/app_request_network.go

```diff
@@ -103,83 +102,99 @@ func NewNetwork(
 		return nil, nil, err
 	}
 
-	// We need to initially connect to some nodes in the network before peer
```

Comment:
There's no need to connect to these beacon nodes, since we explicitly connect to every node that we send `AppRequest` messages to now.